rfcs/remove_ie_edge_legacy.md

```diff
@@ -0,0 +1,32 @@
+# RFC 179: Remove support for Internet Explorer and Microsoft Edge Legacy
+
+### Summary
+
+Remove support for running WPT tests in Internet Explorer and Microsoft Edge
+Legacy.
+
+### Details
+
+Support for the Microsoft Edge Legacy desktop application ended on March 9,
+2021[1], and support for the Internet Explorer desktop application ended on
+June 15, 2022[2].
+
+The proposal is to remove the `edge`, `edge_webdriver`, and `ie` options from
+the product parameter to `wpt run`, along with all related code.
+
+After removal, the remaining supported products would be:
+
+```
+[--product {android_weblayer,android_webview,chrome,chrome_android,chrome_ios,
+chromium,content_shell,edgechromium,firefox,firefox_android,safari,sauce,servo,

```

Comment:
This doesn't need to be decided now, would make sense to wait for a few months after this RFC and then rename `edgechromium` to `edge` to simplify matters.